testscript/doc.go

```diff
@@ -2,320 +2,321 @@
+// 	            # modify p2 - p1 should appear stale (0.029s)
+// 	            # build & install p1 again (0.022s)
+// 	            > go install -i p11
+// 	            [stderr]
+// 	            can't load package: package p11: cannot find package "p11" in any of:
+// 	            	/Users/rsc/go/src/p11 (from $GOROOT)
+// 	            	$WORK/d1/src/p11 (from $GOPATH)
+// 	            	$WORK/d2/src/p11
+// 	            [exit status 1]
+// 	            FAIL: unexpected go command failure
+//
+// 	        script_test.go:73: failed at testdata/script/install_rebuild_gopath.txt:15 in $WORK/gopath/src
+//
+// 	FAIL
+// 	exit status 1
+// 	FAIL	cmd/go	4.875s
+// 	$
+//
+// Note that the commands in earlier phases have been hidden, so that the relevant
+// commands are more easily found, and the elapsed time for a completed phase
+// is shown next to the phase heading. To see the entire execution, use "go test -v",
+// which also adds an initial environment dump to the beginning of the log.
+//
+// Note also that in reported output, the actual name of the per-script temporary directory
+// has been consistently replaced with the literal string $WORK.
+//
+// If Params.TestWork is true, it causes each test to log the name of its $WORK directory and other
+// environment variable settings and also to leave that directory behind when it exits,
+// for manual debugging of failing tests:
+//
+// 	$ go test -run=Script -work
+// 	--- FAIL: TestScript (3.75s)
+// 	    --- FAIL: TestScript/install_rebuild_gopath (0.16s)
+// 	        script_test.go:223:
+// 	            WORK=/tmp/cmd-go-test-745953508/script-install_rebuild_gopath
+// 	            GOARCH=
+// 	            GOCACHE=/Users/rsc/Library/Caches/go-build
+// 	            GOOS=
+// 	            GOPATH=$WORK/gopath
+// 	            GOROOT=/Users/rsc/go
+// 	            HOME=/no-home
+// 	            TMPDIR=$WORK/tmp
+// 	            exe=
+//
+// 	            # GOPATH with p1 in d2, p2 in d2 (0.000s)
+// 	            # build & install p1 (0.085s)
+// 	            # modify p2 - p1 should appear stale (0.030s)
+// 	            # build & install p1 again (0.019s)
+// 	            > go install -i p11
+// 	            [stderr]
+// 	            can't load package: package p11: cannot find package "p11" in any of:
+// 	            	/Users/rsc/go/src/p11 (from $GOROOT)
+// 	            	$WORK/d1/src/p11 (from $GOPATH)
+// 	            	$WORK/d2/src/p11
+// 	            [exit status 1]
+// 	            FAIL: unexpected go command failure
+//
+// 	        script_test.go:73: failed at testdata/script/install_rebuild_gopath.txt:15 in $WORK/gopath/src
+//
+// 	FAIL
+// 	exit status 1
+// 	FAIL	cmd/go	4.875s
+// 	$
+//
+// 	$ WORK=/tmp/cmd-go-test-745953508/script-install_rebuild_gopath
+// 	$ cd $WORK/d1/src/p1
+// 	$ cat p1.go
+// 	package p1
+// 	import "p2"
+// 	func F() { p2.F() }
+// 	$
+//
+// The work directories will be created inside $GOTMPDIR/go-test-script*/script-$scriptname,
```

Comment:
i _think_ it might be better to put this text next to the the docs for `Params.TestWork` (it would be easy to miss, I think).
Also, to avoid the need to change to using `/* */` style for the whole doc comment, how about:

```
The work directories will be created inside $GOTMPDIR/go-test-script$random/script-$scriptname,
where $GOTMPDIR defaults to os.TempDir(), $random is an arbitrary random suffix and $scriptname
is the name of the testscript test file.
```

?